src/utils/webrtc/MediaDevicesManager.js

```diff
@@ -283,13 +283,13 @@ MediaDevicesManager.prototype = {
 	 */
 	listDevices() {
 		navigator.mediaDevices.enumerateDevices().then(devices => {
-			listMediaDevices(
+			console.info(listMediaDevices(
 				devices,
 				this.attributes.audioInputId,
 				this.attributes.videoInputId,
 				this._preferenceAudioInputList,
 				this._preferenceVideoInputList,
-			)
+			))
 		})
 	},
```

Comment:
nit: `_populatePreferences` should not update the browser storage because it doesn't reflect preferences yet, it follows the order of registration

Reply:
This is something to think about: if we won't save device at this moment, and won't join a call to update prefs, device will not be saved anywhere (though it's not really needed, as 'default' will be higher in the list, and be used as a fallback value). But if we pick a device later from available, it will appear in the list eventually.
So might be fine, i think?